.circleci/config.yml

```diff
@@ -317,17 +329,24 @@ workflows:
       # E2E: PWA
       - cypress/run:
           name: 'E2E: PWA'
-          executor: cypress/browsers-chrome76
+          executor: with-chrome
           browser: chrome
           pre-steps:
-            - run: 'rm -rf ~/.yarn && npm i -g yarn && yarn -v && yarn global
-                add wait-on' # Use yarn latest
+            - run: |
+                # Clear yarn cache; update to latest
+                rm -rf ~/.yarn
+                npm i -g yarn
+                yarn -v
+                # Add `wait-on` library for Cypress orb? (global)
+                yarn global add wait-on
           yarn: true
-          record: false
-          store_artifacts: false
+          record: true
+          store_artifacts: true
           working_directory: platform/viewer
-          build: npx cross-env QUICK_BUILD=true yarn run build
-          start: yarn run test:e2e:serve
+          build: |
+            # Build our client
+            npx cross-env QUICK_BUILD=true yarn run build
+          start: APP_CONFIG=config/dicomweb-server.js yarn run dev
```

Comment:
Is this just you trying to test, because this is wonky.

Reply:
yes